ui/src/app/services/workflow/workflow.service.ts

```diff
@@ -165,14 +165,14 @@ export class WorkflowService {
     );
   }
 
-  getWorkflowDynamicFormParts(): Observable<DynamicFormParts> {
+  getWorkflowDynamicFormParts(): Observable<DynamicFormPartsResponse> {
     return this.getJobTemplates().pipe(
       mergeMap((jobTemplates) => {
         const sensorParts = WorkflowService.getSensorDynamicFormParts();
         const jobParts = jobTemplates.map((jobTemplate) => {
           return WorkflowService.getJobDynamicFormPart(jobTemplate);
         });
-        return of(DynamicFormPartsFactory.create(sensorParts, jobParts));
+        return of(DynamicFormPartsResponseFactory.create(DynamicFormPartsFactory.create(sensorParts, jobParts), jobTemplates));
```

Comment:
Do we still need `sensorParts` and `jobParts`? I thought all of that information should be now be hard-coded in the respective components

Reply:
Yes, you are right like in previous comment, I would leave it and do it in the next PR